src/vl-map-wms-layer.js

```diff
@@ -50,6 +51,14 @@ export class VlMapWmsLayer extends VlMapLayer {
     return this.getAttribute('data-vl-styles') || '';
   }
 
+  get _sldBody() {
+    return this.getAttribute('data-vl-sld-body') ? this.__sldBodyWithWhitespaceRemoved : '';
+  }
+
+  get __sldBodyWithWhitespaceRemoved() {
+    return this.getAttribute('data-vl-sld-body').replace(/>\s*/g, '>').replace(/\s*</g, '<');
```

Comment:
Is dat een trim dat je doet? Werkt trim() dan ook niet?

Reply:
ahnee, de regexp werkt ook op 'interne' tags precies.

Reply:
trim is dacht ik enkel voor en na een stuk tekst. Hier zijn het spaties in de tekst zelf als ik het goed begrijp.

Reply:
Klopt, dit om het wat compacter te maken om door te sturen.